File: Feitos em C++/2139.cpp
#include<bits/stdc++.h>
using namespace std;
int main() {
	int month,day;
	while(cin>>month>>day){
		if(month==12&&day==25){
			cout<<"E natal!"<<endl;
		}
		else if(month==12&&day==24){
			cout<<"E vespera de natal!"<<endl;
		}
		else if(month==12&&day>25){
			cout<<"Ja passou!"<<endl;
		}
		else{
			if(month==12){
				cout<<"Faltam "<<25-day<<" dias para o natal!"<<endl;
			}
			else if(month==11){
				cout<<"Faltam "<<25+30-day<<" dias para o natal!"<<endl;
			}
			else if(month==10){
				cout<<"Faltam "<<25+30+31-day<<" dias para o natal!"<<endl;
			}
			else if(month==9){
				cout<<"Faltam "<<25+30+31+30-day<<" dias para o natal!"<<endl;
			}
			else if(month==8){
				cout<<"Faltam "<<25+30+31+30+31-day<<" dias para o natal!"<<endl;
			}
			else if(month==7){
				cout<<"Faltam "<<25+30+31+30+31+31-day<<" dias para o natal!"<<endl;
			}
			else if(month==6){
				cout<<"Faltam "<<25+30+31+30+31+31+30-day<<" dias para o natal!"<<endl;
			}
			else if(month==5){
				cout<<"Faltam "<<25+30+31+30+31+31+30+31-day<<" dias para o natal!"<<endl;
			}
			else if(month==4){
				cout<<"Faltam "<<25+30+31+30+31+31+30+31+30-day<<" dias para o natal!"<<endl;
			}
			else if(month==3){
				cout<<"Faltam "<<25+30+31+30+31+31+30+31+30+31-day<<" dias para o natal!"<<endl;
			}
			else if(month==2){
				cout<<"Faltam "<<25+30+31+30+31+31+30+31+30+31+29-day<<" dias para o natal!"<<endl;
			}
			else{
				cout<<"Faltam "<<25+30+31+30+31+31+30+31+30+31+29+31-day<<" dias para o natal!"<<endl;
			}
		}
	}
	return 0;
}

File: Feitos em Python 3/2139.py
while True:
    try:
        ent = input().split()
        month = int(ent[0])
        day = int(ent[1])
        if month==12 and day==25:
            print("E natal!")
        elif month==12 and day==24:
            print("E vespera de natal!")
        elif month==12 and day>25:
            print("Ja passou!")
        else:
            aux=0
            if month==12:
                aux=25-day
            elif month==11:
                aux=25+30-day
            elif month==10:
                aux=25+30+31-day
            elif month==9:
                aux=25+30+31+30-day
            elif month==8:
                aux=25+30+31+30+31-day
            elif month==7:
                aux=25+30+31+30+31+31-day
            elif month==6:
                aux=25+30+31+30+31+31+30-day
            elif month==5:
                aux=25+30+31+30+31+31+30+31-day
            elif month==4:
                aux=25+30+31+30+31+31+30+31+30-day
            elif month==3:
                aux=25+30+31+30+31+31+30+31+30+31-day
            elif month==2:
                aux=25+30+31+30+31+31+30+31+30+31+29-day
            else:
                aux=25+30+31+30+31+31+30+31+30+31+29+31-day
            print("Faltam "+str(aux)+" dias para o natal!")
    except EOFError:
        break